src/components/ContactListItem/ContactListItem.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function ContactListItem({
+  firstName,
+  lastName,
+  company,
+  index,
+}) {
+  return (
+    <li key={index}>
```

Comment:
Try to avoid index as a key. If you have, go for ID

Reply:
Thanks for the hint! :) In a later story I will add unique IDs to the generated user profiles. I will use the IDs instead of the index.